src/en/clouds-maas.md

```diff
@@ -100,32 +100,38 @@ sudo maas-region apikey --username=$PROFILE
 
 Where $PROFILE is to be replaced by the MAAS username.
 
-## Create the Juju controller
+## Creating a controller
 
-You are now ready to create a Juju controller:
+You are now ready to create a Juju controller for cloud 'maas-cloud':
 
 ```bash
 juju bootstrap maas-cloud maas-cloud-controller
 ```
 
-Above, the name given to the new controller was 'maas-cloud-controller'.
-MAAS will allocate a node from its pool to run the controller on. If you want
-to make sure a specific node is used for this, use constraints (see
-[Create a controller with constraints][create-a-controller-with-constraints]).
+Above, the name given to the new controller is 'maas-cloud-controller'. MAAS
+will allocate a node from its pool to run the controller on.
+
+For detailed explanation and examples of the `bootstrap` command see the
```

Comment:
I think this reads better with an 'a', eg: For *a* detailed explanation...

Reply:
Alrighty.